bundles/org.openhab.binding.sony/src/main/resources/ESH-INF/thing/scalar.xml

```diff
@@ -0,0 +1,1704 @@
+		<item-type>String</item-type>
+		<label>Storage Protection</label>
+		<description>The storage protection status</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstorageisregistered" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Registration</label>
+		<description>The storage registration status</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstorageisselfrecorded" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Self Recorded</label>
+		<description>The storage self recorded status</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstorageissqvsupported" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage SQV</label>
+		<description>The storage SQV (standard quality voice) status</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragelun" advanced="true">
+		<item-type>Number</item-type>
+		<label>Storage LUN</label>
+		<description>The storage logical unit number</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragemounted" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Mounted</label>
+		<description>The storage mounted status</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragepermission" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Permission</label>
+		<description>The storage permission status</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstorageposition" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Position</label>
+		<description>The position of the storage (front, back, internal, etc)</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstorageprotocol" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Protocol</label>
+		<description>The storage protocol</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstorageregistrationdate" advanced="true">
+		<item-type>String</item-type>
+		<label>Storage Registration Date</label>
+		<description>The storage registration date</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragesystemareacapacitymb" advanced="true">
+		<item-type>Number</item-type>
+		<label>Storage System Capacity</label>
+		<description>The storage system capacity (in MB)</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="scalarsystemstoragetimesectofinalize" advanced="true">
+		<item-type>Number</item-type>
+		<label>Storage Time to Finalize</label>
+		<description>The time (in seconds) to finalize</description>
+		<state readOnly="true" />
```

Comment:
I'm not how much you would like to make this change, considering the amount of code that you would have to change. But have you considered adding units of measurements? Like in this example changing item-type to `Number:DataAmount` and adding `pattern="%.1f %unit%"` to state.

Reply:
This is supposed to be line 256-259.